docs/getting_started/flows.md

```diff
@@ -31,6 +31,34 @@ In Windmill, both Scripts and Flows consist of two parts:
 - **Logic**, which in case of a Script is the code, and in case of a Flow 
   is the list of Scripts to be executed.
 
+### For-loops
+
+One functionality that is specific to Flows is the ability to wrap a part 
+of its sequence into a loop.
+
+The expected input is a list, and it is usually coming from either previous
+steps (see this [example][hub-flow2]), or from the Flow's inputs. 
+The variable name to iterate over, can be defined in the `Iterator 
+expression` field of the loop.
+
+For example, to iterate over a list provided as the Flow's input, 
+called `myList`, you would use `result.myList` as the iterator expression.
+
+
+:::tip
+Technically speaking, a for-loop is a flow itself, separate from the 
+main flow. It has its own `flow_input`, made of the iterator, i.e. 
+the `flow_input` of the embedding flow.
```

Comment:
composed of the current iteration's value of the iteration and the embedding flow inputs.